hardware/microwave/mw_source_smbv.py

```diff
@@ -92,21 +102,21 @@ def get_limits(self):
         limits = MicrowaveLimits()
         limits.supported_modes = (MicrowaveMode.CW, MicrowaveMode.SWEEP)
 
-        limits.min_power = -145
```

Comment:
Please leave the absolute minimum/maximum here.
See `hardware/microwave/mw_source_smiq.py` from the master branch about how to do configurable limits correctly.